core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -420,32 +420,31 @@ public ServerBuilder service(PathMapping pathMapping, Service<HttpRequest, HttpR
     }
 
     /**
-     * Binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}
+     * Binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}s
      * of the default {@link VirtualHost}.
      *
      * @throws IllegalStateException if the default {@link VirtualHost} has been set via
      *                               {@link #defaultVirtualHost(VirtualHost)} already
      */
     public <T extends ServiceWithPathMappings<HttpRequest, HttpResponse>>
-            ServerBuilder service(T serviceWithPathMapping) {
-        return service(serviceWithPathMapping, Function.identity());
+            ServerBuilder service(T serviceWithPathMappings) {
+        defaultVirtualHostBuilderUpdated();
+        defaultVirtualHostBuilder.service(serviceWithPathMappings);
+        return this;
     }
 
     /**
-     * Decorates and binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}
+     * Decorates and binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}s
      * of the default {@link VirtualHost}.
      *
      * @throws IllegalStateException if the default {@link VirtualHost} has been set via
      *                               {@link #defaultVirtualHost(VirtualHost)} already
      */
     public <T extends ServiceWithPathMappings<HttpRequest, HttpResponse>,
             R extends Service<HttpRequest, HttpResponse>>
-            ServerBuilder service(T serviceWithPathMapping, Function<T, R> decorator) {
-        requireNonNull(serviceWithPathMapping, "serviceWithPathMapping");
-        requireNonNull(serviceWithPathMapping.pathMappings(), "serviceWithPathMapping.pathMappings()");
-        requireNonNull(decorator, "decorator");
-        serviceWithPathMapping.pathMappings()
-                .forEach(pathMapping -> service(pathMapping, decorator.apply(serviceWithPathMapping)));
+    ServerBuilder service(T serviceWithPathMappings, Function<T, R> decorator) {
```

Comment:
Actually, we use this indentation for all other places where type parameter definitions are too long.